### DISCUSSION TITLE
How can I have accesss to  Jenkins main branch update page

### DISCUSSION THREAD

**Discussion by zhililab** [o3de/sig-build#7] [Q&A]
Hi , so happy to see the release of O3DE, I`m extremely interested in o3de build part . How can I have access to  [Jenkins main branch update page](https://jenkins-o3de.agscollab.com/blue/organizations/jenkins/O3DE/activity/?branch=main)

![image](https://user-images.githubusercontent.com/11768073/124689524-3d3d6480-df0b-11eb-9851-5f3ed5a55b0a.png)

**Comment by brianherrera** [comment]
Hi, this is the correct link to the Jenkins main branch page: https://jenkins.build.o3de.org/job/O3DE/job/main/ 

That link is pointing to the incorrect instance. Can you let me know where you found that link so I can correct it?

**Comment by brianherrera** [comment]
Updated incorrect links: https://github.com/o3de/sig-build/pull/8
